packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -219,6 +226,12 @@ export const AutocompleteInput = <T extends AutocompleteInputOptionType>({
     onChange: handleFocusChange,
   })
 
+  const handleClose = () => {
```

Comment:
**suggestion**: reuse it [here](https://github.com/artsy/palette/blob/ad57a6b10f6bdbfda1bd3087b161ddec04ad9b0f/packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx#L215-L217)